static/src/js/app/components/vocab-list-select/VocabListSelect.vue

```diff
@@ -15,92 +20,92 @@
 </template>
 
 <script>
-  import { SET_VOCAB_LIST } from '../../constants';
-  import VocabListEntry from './VocabListEntry.vue';
-  import VocabListSelectedEntry from './VocabListSelectedEntry.vue';
+import { SET_VOCAB_LIST } from "../../constants";
```

Comment:
Let's keep the indentation and single-quotes within Vue/JavaScript, please.

Reply:
Done.